hibernate-core/src/main/java/org/hibernate/dialect/OracleDialect.java

```diff
@@ -624,7 +626,6 @@ public String extractPattern(TemporalUnit unit) {
 			case HOUR -> "to_number(to_char(?2,'HH24'))";
 			case MINUTE -> "to_number(to_char(?2,'MI'))";
 			case SECOND -> "to_number(to_char(?2,'SS'))";
-			case EPOCH -> "trunc((cast(?2 at time zone 'UTC' as date) - date '1970-1-1')*86400)";
```

Comment:
I think we still need to keep this line, the `extractPattern` method is also used in duration arithmetic IIRC.